charts/incubator/cloudflared/README.md

```diff
@@ -1,107 +0,0 @@
-# cloudflared
```

Comment:
While this is autogenerated, you don't need to delete it.
It will get replaced later.